packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -115,6 +116,7 @@ export interface CallState {
     isMuted: boolean;
     isScreenSharingOn: boolean;
     kind: CallKind;
+    localParticipantReactionState: LocalParticipantReactionState;
```

Comment:
Is `LocalParticipantReactionState` needed?
Can this just be 
```suggestion
    localParticipantReactionState?: ReactionState;
```

This currently this reads as: `call.reactionState.reactionState`

Reply:
This actually reads as `call.localParticipantState.reactionState`

Reply:
> This actually reads as `call.localParticipantState`

Its `localParticipantReactionState` not just `localParticipantState`, so if its `localParticipantReactionState.reactionState` its like saying call.reactionState.reactionState

Reply:
I see. shall we then remove the localparticipantreactionstate altogether? It makes sense to just keep ReactionState. If we need any extra state for locals, we can then create one later.